### DISCUSSION TITLE
New achievement system (only tabs for mods)

### DISCUSSION THREAD

**Discussion by kotmatross28729** [Roadhog360/Et-Futurum-Requiem#185] [Ideas]
In general, I noticed that on 1.7.10 (well, or before some version there) a very bad menu of achievements, in new versions there are special tabs to switch (I'm not talking about vanilla tabs, for example nether, end, I say what if you have mods installed, then tabs are created there, between which you can switch very easily) but in older versions, if you have mods installed, then you need to press one button several times and wait until the branch of the desired mod drops out, and if mods a lot, then you can click again, and you have to click all over again

![ff32117fa68d45219f037e361af55a97](https://user-images.githubusercontent.com/110309314/194740331-9977f8f6-c024-4ab1-8a53-d865bc0a0479.png)

**Comment by SilearFlare** [comment]
BetterArchievements[ already does this](https://www.curseforge.com/minecraft/mc-mods/betterachievements) and is immensely popular. 

Let's not try to reinvent the wheel here.

**Reply by kotmatross28729** [reply]
> BetterArchievements[ already does this](https://www.curseforge.com/minecraft/mc-mods/betterachievements) and is immensely popular.
> 
> Let's not try to reinvent the wheel here.

Wow, thanks.
To be honest, I didn't even know it existed.
